packages/syntax-floating-components/src/Popover/Popover.stories.tsx

```diff
@@ -1,13 +1,115 @@
+      laborum.{" "}
+    </Typography>
+    <Typography tooltip="This is a tooltip">
+      Daily maximum of 30 minutes. And a lot of really long text. Lorem ipsum
+      dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+      incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis
+      nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+      consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+      cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
+      non proident, sunt in culpa qui officia deserunt mollit anim id est
+      laborum.{" "}
+    </Typography>
+    <Typography tooltip="This is a tooltip">
+      Daily maximum of 30 minutes. And a lot of really long text. Lorem ipsum
+      dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+      incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis
+      nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+      consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+      cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
+      non proident, sunt in culpa qui officia deserunt mollit anim id est
+      laborum.{" "}
+    </Typography>
+    <Typography tooltip="This is a tooltip">
+      Daily maximum of 30 minutes. And a lot of really long text. Lorem ipsum
+      dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+      incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis
+      nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+      consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+      cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
+      non proident, sunt in culpa qui officia deserunt mollit anim id est
+      laborum.{" "}
+    </Typography>
+    <Typography tooltip="This is a tooltip">
+      Daily maximum of 30 minutes. And a lot of really long text. Lorem ipsum
+      dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+      incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis
+      nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+      consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+      cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
+      non proident, sunt in culpa qui officia deserunt mollit anim id est
+      laborum.{" "}
+    </Typography>
+  </>
+);
 
 export default {
   title: "Floating-Components/Popover",
   component: Popover,
+  decorators: [
+    (Story) => (
+      <Box
+        width="calc(100vw - 2rem)"
+        height="calc(100vh - 2rem)"
+        backgroundColor="gray100"
+        display="flex"
+        direction="column"
+        justifyContent="center"
+        alignItems="center"
+      >
+        <Story />
+      </Box>
+    ),
+  ],
```

Comment:
@benjroy let's make sure all the properties listed here are still available on the component - e.g. `onOpenChange` / `strategy`